templates/components/products/card.html

```diff
@@ -81,7 +81,11 @@ <h4 class="card-title">
 
         <div class="card-text" data-test-info-type="price">
             {{#or customer (if theme_settings.restrict_to_login '!==' true)}}
-                {{> components/products/price price=price customer=customer}}
+                {{#if price_range}}
```

Comment:
what about product view ? https://github.com/bigcommerce/cornerstone/blob/master/templates/components/products/product-view.html#L19 are we not planning on showing the range in the quick view & product detail page until an option is selected ?

Reply:
From what I understand, this is only for PLP and nowhere else.